pkg/rpm/salt-minion

```diff
@@ -70,7 +68,8 @@ _su_cmd() {
 
 _get_pid() {
     netstat $NS_NOTRIM -ap --protocol=unix 2>$ERROR_TO_DEVNULL \
-        | sed -r  -e "\|\s${SOCK_DIR}/minion_event_${MINION_ID_HASH}_pub\.ipc$|"'!d; s|/.*||; s/.*\s//;'
+        | sed -r  -e "\|\s${SOCK_DIR}/minion_event_${MINION_ID_HASH}_pub\.ipc$|"'!d; s|/.*||; s/.*\s//;' | \
+        head -n 1
```

Comment:
This is clearly my mistake, I didn't read the whole script carefully enough. Thanks for spotting it!

The reason for this change was that the `_get_pid()` always return the same PID twice:
```sh
netstat --notrim -ap --protocol=unix 2>/dev/null | \
sed -r  -e "\|\s/var/run/salt/minion/minion_event_8af36c78bc_pub\.ipc$|"'!d; s|/.*||; s/.*\s//;'
932
932
```

And that was causing a little bit strange `status` output.
```
/etc/init.d/salt-minion status
salt-minion:root:c5-sysv is running: 932
932
```

I should use `uniq` here.